Elf/__init__.py

```diff
@@ -127,6 +139,7 @@ def __init__(self, fd, inmem=False):
         except Exception as e:
             logger.warning("Exception parsing Dynamics from Sections: %r" % e, exc_info=1)
 
+>>>>>>> master
```

Comment:
this concerns me :)  please review

Reply:
I mean, I told you it wasn't ready, but you asked for it as is so you can see where it's at :stuck_out_tongue:

Reply:
lol!   <facepalm> :stuck_out_tongue_closed_eyes: